.github/workflows/build-gem.yml

```diff
@@ -0,0 +1,21 @@
+name: Check if the gem exists on rubygems.org
+
+on: [pull_request]
+
+jobs:
+  build-gem:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.6
+      - name: Check if gem exists
+        run: |
+          GEM_VERSION=$(ruby -e "require 'scc/codestyle'; print Scc::Codestyle::VERSION" -I lib)
+          # if we install the gem, then the $GEM_VERSION already exist on rubygems.org
+          # therefore error
+          gem install -q --silent scc-codestyle -v $GEM_VERSION || (echo "Please update the gem version" && exit 0)
```

Comment:
We can use [Rubygems API](https://guides.rubygems.org/rubygems-org-api/#gem-version-methods) to return the latest version of the gem instead of installing it. I believe that this is going to be even faster.

Reply:
>  I believe that this is going to be even faster.

hey @arharovets yes, well if the gem doesn't  exist (the happy path, when developer bumped the gem version accordingly), both solutions would have same execution time.  Maybe could you provide a one-liner that mimic `gem install` (I'm assuming some `curl` piping to `jq` setting the exit code as needed?) to be used here? Otherwise I don't think some custom code would justify the work, specially that it is already a fast step (under 14s).

Reply:
we could probably reduce this time by half, I'm experimenting on that. However, it shouldn't block your PR. Thank you for the really good work on this one!